packages/core/src/Lib/UseClass/Internal/Lib/UseComponentClass.ts

```
@@ -0,0 +1,13 @@
+import type { ComponentName }                 from '@/Lib';
+import type { ComputedRef, MaybeRefOrGetter } from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Lib" (internal-type).

(perfectionist/sort-imports)